weaviate/templates/_helpers.tpl

```diff
@@ -181,6 +181,15 @@ Raft cluster configuration settings
 {{- define "raft_configuration" -}}
   {{- $replicas := .Values.replicas | int -}}
   {{- $voters := .Values.env.RAFT_BOOTSTRAP_EXPECT | int -}}
+  {{- if empty .Values.env.RAFT_BOOTSTRAP_EXPECT -}}
+    {{- if ge $replicas 10 -}}
+      {{- $voters = 5 -}}
+    {{- else if ge $replicas 4 -}}
```

Comment:
I would change this into `else if ge $replicas 3` as it makes more sense to have a 3 node cluster with 3 voters than with a single 1, doesn't it? Then if the cluster replicas is smaller than 3, we go for 1 voter. what do you think?

Reply:
Yes, great point, I will change this

Reply:
Fixed